Geometry/MuonCommonData/test/BuildFile.xml

```diff
@@ -0,0 +1,4 @@
+<bin file="dummyMain.cpp" name="GeometryMuonCommonDataTestDriver">
```

Comment:
@ianna the suggestion by @makortel adopted in https://github.com/cms-sw/cmssw/pull/29841/files#diff-beba013c8309607c0eb9e70dad9fc446 was slightly different, I do not have a particularly strong feeling about this, and the TEST_RUNNER approach allows the developer to use some variables. But probably the situation is not that different (here and elsewhere in this PR).

Reply:
yes, thanks!